docs/wfc3tools/wf3cte.rst

```diff
@@ -76,7 +76,9 @@ PERFORM, then the CTE correction will be applied to the dataset. Some caveats fo
 
 .. note::
 
-   The WFC3 CTE algorithm was updated in `calwf3`/`wf3cte` v3.6.0 (December 2020) where the pixel-based model is better constrained for both low and high pixel values.  See the thorough discussion regarding the update in `ISR WFC3 2021-09 <https://innerspace.stsci.edu/display/WFC3RSRC/Instrument+Science+Reports?preview=/90064318/272889913/WFC3-ISR-2021-09.pdf>`_.
+   The WFC3 CTE algorithm was updated in `calwf3`/`wf3cte` v3.6.0 (December 2020) where the pixel-based model is better constrained for both low and high pixel values.  See the thorough discussion regarding the update in `WFC3 ISR 2021-09 
+ <https://www.stsci.edu/files/live/sites/www/files/home/hst/ instrumentation/wfc3/documentation/instrument-science-reports-isrs/_documents/2021/WFC3-ISR-2021-09.pdf>`_.
```

Comment:
There's a space in the hyperlink (probably an artifact of copy/paste).
.../hst/ instrumentation/... should be .../hst/instrumentation/...

Reply:
Indeed! Fixed in 0310031.